.gitignore

```diff
@@ -7,4 +7,7 @@
 .project
 .pydevproject
 wsgi/media/photologue/photos/
-wsgi/odyssy/.idea/
+wsgi/odyssy/
```

Comment:
No there are not .idea files in current codebase and I think there is no need to upload them.